### ISSUE TITLE
ArgumentOutOfRangeException when replacing Text

### ISSUE THREAD

**Issue by milefork** [xceedsoftware/DocX#312]
Hi folks,

I got an error with DocX.
I tried to replace placeholders with text. While doing that an ArgumentOutOfRangeException comes up reproduceable.
Here is my code: https://github.com/milefork/DocxErrorSample

Greetings

**Comment by Rapiiidooo** [comment]
Downgrade to version 1.2.1 will fix temporary.

**Comment by XceedBoucherS** [comment]
Hi,
Thank you for your sample.
This is already fixed in v1.5 and up.
You can test the latest version v1.6 for free for 45 days here : https://xceed.com/xceed-words-for-net/.
Not sure why it didn't work in your case. I tried removing the DocX reference from Nuget et re-add it. It worked.

**Comment by milefork** [comment]
@Rapiiidooo yeah in this sample downgrading fixed it temporary. But in my real case project the error will come in another row with the downgraded version.

@XceedBoucherS sorry but the paid version is not an option for me.
I think your redadding case worked because the previous run succeed and the document was already modified.

**Comment by JoeCoDev** [comment]
I ran into this error today as well.  Rolling back to 1.2.1, or forward to 1.6 (I was initially on 1.5).
I found that the issue was a column break immediately before one of the ReplaceText keys.

Sample code:
`            Dictionary<string, string> keyval = new Dictionary<string, string>();
            keyval.Add("Key1", "1");
            keyval.Add("Key2", "2");
            keyval.Add("Key3", "3");

            var sampleDoc = DocX.Load(@".\SampleFile.docx");

            foreach (var item in keyval)
            {
                sampleDoc.ReplaceText(item.Key, item.Value);      // Exception here
            }

            sampleDoc.Save();
`

[SampleFile.docx](https://github.com/xceedsoftware/DocX/files/3989963/SampleFile.docx)

Work-around was to reformat the document so the exception does not occur.

**Comment by XceedBoucherS** [comment]
Hi,
Thank you for reporting this.
Yes, this problem occurs because of the column break preceding a text to replace.
In v1.7, you won't see the ArgumentOutOfRangeException anymore. But the text preceding the column break won't be completely replaced (the new text will be added but the old one won't be removed).
In v1.8, this will all be fixed.

Thank you.
